bin/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Compiled class file
```

Comment:
Please add the entire bin folder to .gitignore (the gitignore in the main repo folder)